src/assets/wufo-players.js

```diff
@@ -0,0 +1,209 @@
+export const wufoPlayers = [
+  {
+    "isCaptain": true,
+    "name": "Ben Siciliano",
+    "nickname": "Italian Stallion",
+    "year": "Senior",
+    "number": 13,
+    "position": "Cutter",
+    "hometown": "Queens, NY"
```

Comment:
I think we should add pronouns for all the players too. There's definitely space for it on the back of the card — I'm thinking sandwiched between `Nickname` and `Year`